frontend/src/features/workspace/components/AdminSwitchEnvMessage.tsx

```diff
@@ -44,7 +44,7 @@ export const AdminSwitchEnvMessage = (): JSX.Element => {
             <VisuallyHidden>
               Click to switch to the original FormSG
             </VisuallyHidden>
-            <Text as="span" display="inline" aria-hidden>
+            <Text as="span" display="inline" aria-hidden cursor="pointer">
```

Comment:
should we add the cursor to the button instead of the text element itself? but buttons are supposed to have innate cursors, not sure why it is not showing up 🤷

Reply:
That was what I thought as well. In fact when analysing the Chakra UI [docs](https://chakra-ui.com/docs/components/button) it seems that the `cursor: pointer` attribute is explicitly set for all buttons. My best guess is that `as="u"` causes the component to behave differently. 

Hopefully, setting `cursor="pointer"` does the trick.

<img width="728" alt="Screenshot 2022-10-13 at 1 37 59 PM" src="https://user-images.githubusercontent.com/34817364/195510705-7583a35e-326c-4273-8e57-18a41923ebd5.png">